src/export.rs

```diff
@@ -64,13 +62,11 @@ impl Priority {
     }
 }
 
-#[cfg(feature = "nightly")]
```

Comment:
This change bumps the MSRV from 1.31.0 to 1.35.0 (or 36) so it cannot be done in the v0.4.x line -- it's a breaking change.

Reply:
Agreed, but the current implementation is UB - then the only choice is to yank all of 0.4.x IMO :/

Reply:
There's another choice: add an opt-in `core-maybeuninit` Cargo feature that switches the uses of MaybeUninit to the `core` version but also bumps the MSRV to 1.35 (or 36, I forget). The panic message can then be changed to point to that feature, which works on a recent stable release, instead of `nightly`. This option is not a breaking change.

Reply:
Hmm, this we probably can do.
It seems strange though to allow this to exist for stable as we are clearly hitting UB - all of `0.4.x` should be yanked the second we release `0.5.0` or we can do another release of `0.4.x` and remove the stable support so people can continue to run `0.4.x` safely.

I think these are our only 2 options. Thoughts @japaric ?